Yoast/wordpress-seo
# Adds the feature toggle for the Recalibration beta.

## Summary

This PR can be summarized in the following changelog entry:

* Adds a toggle feature for subscribing to the recalibration beta.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Go the SEO - Features
* Set the recalibration beta to 'on'
* Go to the post overview and see you have a 'stale content filter'
* Edit a post and verify (javascript console - network) that the analysis.js file has been loaded from my.yoast.com
* The analysis-worker should be loaded from `my-yoast-proxy.php?file=research-webworker`

Note: The newsletter subscription endpoint will be implemented in a separate issue: #11592 

## Quality assurance


Fixes #11450
Makes sure you merge this pull before merging #11595
